src/MF_OutputShifter/MFOutputShifter.cpp

```diff
@@ -19,7 +19,11 @@ void MFOutputShifter::setPin(uint8_t pin, uint8_t value, uint8_t refresh)
     uint8_t idx = (pin & 0xF8) >> 3;
     uint8_t msk = (0x01 << (pin & 0x07));
 
+#ifdef REVERSED_OUTPUT_OUTPUTSHIFTER
+    if (value == 0) {
+#else
     if (value > 0) {
```

Comment:
Could this not be like

```cpp
if (value != MF_LOW)
```

And we would not need the `#ifdef` `#else`

Reply:
Hmhm, MF_LOW is only defined for the outputs within MFOutput.h.

Additionally I just recognized that the clear() function must also be adapted

Reply:
Yeah, it makes absolutely sense.
Added the defines to MFOutputShifter.h, otherwise I had to include MFOutput.h or add the defines to all MFBoards.h (which might be also a solution)